_layouts/guides.html

```diff
@@ -51,6 +52,9 @@
 <div class="guide">
   <div class="grid-wrapper">
     <div class="grid__item width-8-12 width-12-12-m">
+      {% if docversion == 'latest' %}
```

Comment:
@gastaldi I just realized that you also have a "vitual" `main` version (`version == 'main'`). Not sure if that was added after I started this PR or whether I just didn't notice...Anyway, I guess it would make sense to also display the edit link for the main version, wouldn't it?

Reply:
Right, AFAIK submitting a PR to a doc will affect the `Main - SNAPSHOT`version, so +1 to that

Reply:
Thanks, fixed with #1984.